src/pe/header.rs

```diff
@@ -29,6 +80,31 @@ impl DosHeader {
                 signature
             )));
         }
+
+        let bytes_on_last_page = bytes.gread_with(&mut offset, scroll::LE)?;
+        let pages_in_file = bytes.gread_with(&mut offset, scroll::LE)?;
+        let relocations = bytes.gread_with(&mut offset, scroll::LE)?;
+        let size_of_header_in_paragraphs = bytes.gread_with(&mut offset, scroll::LE)?;
+        let minimum_extra_paragraphs_needed = bytes.gread_with(&mut offset, scroll::LE)?;
+        let maximum_extra_paragraphs_needed = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_relative_ss = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_sp = bytes.gread_with(&mut offset, scroll::LE)?;
+        let checksum = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_ip = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_relative_cs = bytes.gread_with(&mut offset, scroll::LE)?;
+        let file_address_of_relocation_table = bytes.gread_with(&mut offset, scroll::LE)?;
+        let overlay_number = bytes.gread_with(&mut offset, scroll::LE)?;
+        let reserved = [0x0; 4];
+        offset += core::mem::size_of::<[u16; 4]>();
+        //let reserved = bytes.gread_with(&mut offset, scroll::LE)?;
+        let oem_id = bytes.gread_with(&mut offset, scroll::LE)?;
+        let oem_info = bytes.gread_with(&mut offset, scroll::LE)?;
+        //let reserved2 = bytes.gread_with(&mut offset, scroll::LE)?;
+        let reserved2 = [0x0; 10];
+        offset += core::mem::size_of::<[u16; 10]>();
```

Comment:
```suggestion
        offset += core::mem::size_of_val(&reserved2);
```

Reply:
Isn't `size_of_val` nightly-only? cc https://github.com/rust-lang/rust/issues/46571

Reply:
no, it's in since 1.0.0 but the const attribute on it is only available on unstable. (am I reading this wrong? it works here on stable: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=4656199ffa04071b1675b0ad3106a0d4)